Source/SPTablesList.m

```diff
@@ -1740,6 +1765,8 @@ - (BOOL)selectionShouldChangeInTableView:(nullable NSTableView *)aTableView
  */
 - (void)tableViewSelectionDidChange:(NSNotification *)aNotification
 {
+	if (!isMenuOpened) lastSelectedRows = [tablesListView selectedRowIndexes];
+
```

Comment:
I'm a little confused, why do we need to track if the menu is open? Perhaps we should remember which table is selected distinctly from what row is selected? I don't really like the tracking-menu-is-open thing

Reply:
Yes I think you right